docker/measure-worker/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM gcr.io/fuzzbench/base-runner
+
+ENV WORK /work
+WORKDIR $WORK
+
+ENV VIRTUALENV_DIR $WORK/.venv
```

Comment:
nit: good to keep all ENVs at top.

Reply:
I don't agree. I think defining variables close to where they are used is most readable.